src/EditorFeatures/CSharp/MisplacedUsings/MisplacedUsingsCodeFixProvider.IndentationSettings.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis.Formatting;
+using Microsoft.CodeAnalysis.Options;
+
+namespace Microsoft.CodeAnalysis.CSharp.MisplacedUsings
+{
+    /// <summary>
+    /// Implements a code fix for all misplaced using statements.
+    /// </summary>
+    internal partial class MisplacedUsingsCodeFixProvider
+    {
+        internal class IndentationSettings
+        {
+            public int IndentationSize { get; }
+            public int TabSize { get; }
+            public bool UseTabs { get; }
```

Comment:
this seems wacky to pass along when we have an Options type that holds this sort of thing.

Reply:
This was an artifact of the original implementation not having access to .editorconfig. Code using this type could be updated in Roslyn to use the .editorconfig values instead.